assets/js/components/Collection/Search.jsx

```diff
@@ -1,19 +1,25 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Link } from "react-router-dom";
 
-const CollectionSearch = () => {
-  let items = [];
+const CollectionSearch = ({ collection }) => {
+  const [filteredWorks, setFilteredWorks] = useState(collection.works);
+  // useEffect(() => {
+  //   setFilteredWorks(collection ? collection.works : []);
+  // }, []);
 
-  for (let i = 0; i < 20; i++) {
-    items.push(
-      <li className="column is-one-quarter-desktop is-half-tablet" key={i}>
-        <figure className="image is-square">
-          <img src="/images/480x480.png" />
-        </figure>
-        <p className="text-center">{`Image title ${i} `}</p>
-      </li>
-    );
-  }
+  const handleFilterChange = (e) => {
+    const filterValue = e.target.value.toUpperCase();
+    if (!filterValue) {
+      return setFilteredWorks(collection ? collection.works : []);
+    }
+    const filteredList = collection.works.filter((work) => {
+      return work.descriptiveMetadata.title
+        ? work.descriptiveMetadata.title.toUpperCase().indexOf(filterValue) > -1
+        : false;
```

Comment:
This needs to be false value. `false` here means that it doesn't match so it will not be added to the filteredList. If we add `[]` here, it returns `true` which means the item is a match.

Reply:
Ah yes, you're right.  I confused my returns reading this, sorry about that.